### PR TITLE
OSDOCS-8515: Documented the 4.15 Welcome page

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.15

Issue:
* [OSDOCS-8515](https://issues.redhat.com/browse/OSDOCS-8515)

Link to docs preview:
[Welcome page](https://70120--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/)

QE review:
- [ ] QE has approved this change.

Additional information:
* [Google document](https://docs.google.com/document/d/1UTtAuYfRjOEyVw-rh7J7GCmY8WPgH_9ICTW1l3HWPsU/edit)

### CODE REVIEW

**Comment by dfitzmau** [comment]
/label peer-review-needed

Note to reviewers. Most feedback was pooled from the [Google Doc](https://docs.google.com/document/d/1UTtAuYfRjOEyVw-rh7J7GCmY8WPgH_9ICTW1l3HWPsU/edit#heading=h.othf45wpltt).

Any items in the GDoc coloured in Orange might or might not make the 4.15 release. A new PR could be created closer to that time by component writers to update the Welcome page.

File: welcome/index.adoc [hunk 1]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
-endif::openshift-origin[]
+- **xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Installing {product-title} with the Agent-based Installer]**: You can use the Agent-based Installer to generate a bootable ISO image that contains the Assisted discovery agent, the Assisted Service, and all the other information required to deploy an {product-title} cluster. The Agent-based Installer leverages the advantages of the Assisted Installer in a disconnected environment
+
+- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using the bare metal user-provisioned infrastructure route.
```
Comment (mburke5678) [inline welcome/index.adoc:95]: _Route_ seems a bit marketing-speak to me. Not sure if this suggestion is accurate, though.

```suggestion
- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using bare metal user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 2]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
+
+- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
 ifndef::openshift-origin[]
-- **xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Install a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}]**: You can install {product-title} with z/VM on {ibm-z-name} and {ibm-linuxone-name} on user-provisioned infrastructure.
+- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on {ibm-cloud-name}]**: On {ibm-cloud-name}, you can install {product-title}  on installer-provisioned infrastructure.
```
Comment (mburke5678) [inline welcome/index.adoc:100]:
```suggestion
- **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on {ibm-cloud-name}]**: On {ibm-cloud-name}, you can install {product-title} on installer-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 3]
```diff
@@ -277,29 +284,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
...
+- **xref:..//updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
 Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster. If an update is available from the OpenShift Update Service (OSUS), you apply that cluster update from the {product-title} xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[web console] or the xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[OpenShift CLI] (`oc`).
 
-- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[Using the OpenShift Update Service in a disconnected environment]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in disconnected environments.
+- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/index.html[Using the OpenShift Update Service in a disconnected environment]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in disconnected environments.
```
Comment (mburke5678) [inline welcome/index.adoc:296]: Should this one also use the _You can install_ structure like most/all of the other links here?

File: welcome/index.adoc [hunk 4]
```diff
@@ -277,29 +284,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
...
 
 - **xref:../nodes/clusters/nodes-cluster-worker-latency-profiles.adoc#nodes-cluster-worker-latency-profiles[Improving cluster stability in high latency environments by using worker latency profiles]**: If your network has latency issues, you can use one of three worker latency profiles to help ensure that your control plane does not accidentally evict pods in case it cannot reach a worker node. You can configure or modify the profile at any time during the life of the cluster.
 
-=== Monitor the cluster
+=== Obeserve a cluster
```
Comment (mburke5678) [inline welcome/index.adoc:300]:
```suggestion
=== Observe a cluster
```

File: welcome/index.adoc (hunk 3 above)
Comment (mburke5678) [reply welcome/index.adoc:296]: Should this one also use the _You can_ structure like most/all of the other links here? 

```suggestion
- **xref:../updating/updating_a_cluster/updating_disconnected_cluster/index.html[Using the OpenShift Update Service in a disconnected environment]**:  You can use the OpenShift Update Service for recommending {product-title} updates in disconnected environments.
```

File: welcome/index.adoc [hunk 5]
```diff
@@ -277,29 +284,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
...
+
+- **xref:../logging/cluster-logging.adoc#cluster-logging[OpenShift Logging]**: Learn about logging and configure different logging components, such as log storage, log collectors, and the logging web console plugin.
+
+- **xref:../distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc[Red Hat OpenShift distributed tracing platform]**: store and visualize large volumes of requests passing through distributed systems, across the whole stack of microservices, and under heavy loads. Use it for monitoring distributed transactions, gathering insights into your instrumented services, network profiling, performance and latency optimization, root cause analysis, and troubleshooting the interaction between components in modern cloud-native microservices-based applications.
```
Comment (mburke5678) [inline welcome/index.adoc:304]:
```suggestion
- **xref:../distr_tracing/distr_tracing_arch/distr-tracing-architecture.adoc[Red Hat OpenShift distributed tracing platform]**: Store and visualize large volumes of requests passing through distributed systems, across the whole stack of microservices, and under heavy loads. Use the distributed tracing platform for monitoring distributed transactions, gathering insights into your instrumented services, network profiling, performance and latency optimization, root cause analysis, and troubleshooting the interaction between components in modern cloud-native microservices-based applications.
```

File: welcome/index.adoc [hunk 6]
```diff
@@ -277,29 +284,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
...
+- **xref:../otel/otel-release-notes.adoc[Red Hat build of OpenTelemetry]**: Instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior. Use open source backends like Tempo or Prometheus, or use commercial offerings. Learn a single set of APIs and conventions, and own the data that you generate.
 
-- **xref:../monitoring/monitoring-overview.adoc#monitoring-overview[Monitor]**:
+// CHECK IN CLOSER TO THE GA DATE BECAUSE FEATURE WORK IS NOT MERGED; ESTIMATED MERGE BY 19 FEB. ADD xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-viewing-dashboards_metrics-dashboards-alerts to [view dashbaords text.
+- **xref:../network_observability/network-observability-overview.adoc[Network Observability]**: Observe network traffic for OpenShift Container Platform clusters using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```
Comment (mburke5678) [inline welcome/index.adoc:309]: Is there a reason not to use {product-title}?
```suggestion
- **xref:../network_observability/network-observability-overview.adoc[Network Observability]**: Observe network traffic for (produt-title} clusters using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```

Comment (mburke5678) [reply welcome/index.adoc:309]:
```suggestion
- **xref:../network_observability/network-observability-overview.adoc[Network Observability]**: Observe network traffic for OpenShift Container Platform clusters by using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```

**Comment by mburke5678** [comment]
@dfitzmau A few comments, otherwise LGTM

File: welcome/index.adoc [hunk 7]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
 
-- **xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Install a cluster on VMware vSphere]**: You can install {product-title} on supported versions of vSphere.
+- **xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[Install a cluster on {ibm-power-name}]**: On {ibm-power-name}, you can install {product-title} on user-provisioned infrastructure.
 
-- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using the bare metal user-provisioned infrastructure route.
+- **xref:../installing/installing_ibm_z/installing-ibm-z.html#installing-ibm-z[Install a cluster on {ibm-z-name} and {ibm-linuxone-name}]**: On {ibm-z-name} and {ibm-linuxone-name}, you can install {product-title} on user-provisioned infrastructure.
```
Comment (SNiemann15) [inline welcome/index.adoc:106]:
```suggestion
- **xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Install a cluster on {ibm-z-name} and {ibm-linuxone-name}]**: On {ibm-z-name} and {ibm-linuxone-name}, you can install {product-title} on user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 8]
```diff
@@ -124,8 +129,7 @@ ifndef::openshift-origin[]
...
+xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere], xref:../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[{ibm-z-name} and {ibm-linuxone-name} with z/VM], xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[{ibm-z-name} and {ibm-linuxone-name} with RHEL KVM], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
```
Comment (SNiemann15) [inline welcome/index.adoc:132]:
```suggestion
xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere], xref:../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[{ibm-z-name} and {ibm-linuxone-name}], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
```

File: welcome/index.adoc [hunk 9]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
 
 - **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack Hub, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
-- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
+- **xref:../installing/installing_on_prem_assisted/installing-on-prem-assisted.html[Install a cluster by using the Assisted Installer]**: The Assisted Installer is an installation solution that is provided on the Red Hat Hybrid Cloud Console. The Assisted Installer supports installing an {product-title} cluster on many platforms, but with a focus on bare metal, Nutanix, and vSphere infrastructures.
```
Comment (jeana-redhat) [inline welcome/index.adoc:87]: vSphere has an attribute too :)
```suggestion
- **xref:../installing/installing_on_prem_assisted/installing-on-prem-assisted.html[Install a cluster by using the Assisted Installer]**: The Assisted Installer is an installation solution that is provided on the Red Hat Hybrid Cloud Console. The Assisted Installer supports installing an {product-title} cluster on many platforms, but with a focus on bare metal, Nutanix, and {vmw-short} infrastructures.
```

File: welcome/index.adoc [hunk 10]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
 
-- **xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[Install a cluster on {ibm-power-name}]**: You can install {product-title} on {ibm-power-name} on user-provisioned infrastructure.
-endif::openshift-origin[]
+- **xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Installing {product-title} with the Agent-based Installer]**: You can use the Agent-based Installer to generate a bootable ISO image that contains the Assisted discovery agent, the Assisted Service, and all the other information required to deploy an {product-title} cluster. The Agent-based Installer leverages the advantages of the Assisted Installer in a disconnected environment
```
Comment (jeana-redhat) [inline welcome/index.adoc:93]: These two seem to be duplicates of the two above, with slightly different titles:
image

File: welcome/index.adoc [hunk 11]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
+
+- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using bare metal user-provisioned infrastructure.
+
+- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```
Comment (jeana-redhat) [inline welcome/index.adoc:97]:
```suggestion
- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on {gcp-short}]**: On {gcp-first} you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
```

File: welcome/index.adoc [hunk 12]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
 
-- **xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#preparing-to-install-on-bare-metal[Install a cluster on bare metal]**: On bare metal, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure. If none of the available platform and cloud provider deployment options meet your needs, consider using the bare metal user-provisioned infrastructure route.
+- **xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Install a cluster on {ibm-z-name} and {ibm-linuxone-name}]**: On {ibm-z-name} and {ibm-linuxone-name}, you can install {product-title} on user-provisioned infrastructure.
+endif::openshift-origin[]
+////
+// Addition will be tracked through https://issues.redhat.com/browse/OSDOCS-9514
+- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}, so that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
```
Comment (jeana-redhat) [inline welcome/index.adoc:110]: Consider breaking up the long sentence.
```suggestion
- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This allows you to run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
```

File: welcome/index.adoc [hunk 13]
```diff
@@ -84,32 +84,37 @@ Explore the following {product-title} installation tasks:
...
 
-- **Install a cluster on {rh-openstack-first}**: You can install a cluster on {rh-openstack} with customizations or on a restricted network on installer-provisioned infrastructure. For more information, see the following documentation:
-** xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[Installing a cluster on {rh-openstack} with customizations]
-** xref:../installing/installing_openstack/installing-openstack-installer-restricted.adoc#installing-openstack-installer-restricted[Installing a cluster on {rh-openstack} in a restricted network]
+- **xref:../installing/installing_openstack/preparing-to-install-on-openstack.adoc#preparing-to-install-on-openstack[Install a cluster on {rh-openstack-first}]**: On {rh-openstack}, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
+- **xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Install a cluster on VMware vSphere]**: You can install {product-title} on supported versions of vSphere.
```
Comment (jeana-redhat) [inline welcome/index.adoc:117]:
```suggestion
- **xref:../installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Install a cluster on {vmw-full}]**: You can install {product-title} on supported versions of {vmw-short}.
```

File: welcome/index.adoc [hunk 14]
```diff
@@ -124,8 +129,7 @@ ifndef::openshift-origin[]
...
-xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere],
-xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[{ibm-z-name} and {ibm-linuxone-name} with z/VM], xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[{ibm-z-name} and {ibm-linuxone-name} with RHEL KVM], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
+xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[vSphere], xref:../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[{ibm-z-name} and {ibm-linuxone-name}], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
```
Comment (jeana-redhat) [inline welcome/index.adoc:132]:
```suggestion
xref:../installing/installing_vsphere/upi/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[{vmw-short}], xref:../installing/installing_ibm_cloud_public/installing-ibm-cloud-restricted.adoc#installing-ibm-cloud-restricted[{ibm-cloud-name}], xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[{ibm-z-name} and {ibm-linuxone-name}], xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power[{ibm-power-name}],
```

File: welcome/index.adoc [hunk 15]
```diff
@@ -208,8 +211,11 @@ xref:../applications/working_with_helm_charts/understanding-helm.adoc#understand
 xref:../operators/operator_sdk/helm/osdk-helm-support.adoc#osdk-helm-support[Helm], or configure xref:../operators/operator_sdk/osdk-monitoring-prometheus.adoc#osdk-monitoring-prometheus[built-in Prometheus monitoring] by using the Operator SDK.
 
 - **Reference the xref:../rest_api/index.adoc#api-index[REST API index]**: Learn about {product-title} application programming interface endpoints.
-endif::[]
-endif::[]
+
+// Need to provide a link closer to 4.15 GA
+- **Software Supply Chain Enhancements**: The PipelineRun *Details* view in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of PipelineRuns within a Project. Learn about the Software Supply Chain enhancements and how to deploy them.
```
Comment (jeana-redhat) [inline welcome/index.adoc:216]: - "PipelineRun" looks like an object of some type. Probably needs backticks and some noun to follow it. 
- Should also double-check "Software Supply Chain" capitalization. Is it a component, or should that just be normal lower case use?

```suggestion
- **Software Supply Chain enhancements**: The `PipelineRun` *Details* view in the *Developer* or *Administrator* perspective of the web console provides an enhanced visual representation of `PipelineRun` objects within a project. Learn about the Software Supply Chain enhancements and how to deploy them.
```

File: welcome/index.adoc [hunk 16]
```diff
@@ -277,29 +283,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
...
 - **xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]**: Choose from CPU, memory, and other system resources to xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas].
-endif::[]
+endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 - **xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]**: Reclaim space by pruning unneeded Operators, groups, deployments, builds, images, registries, and cron jobs.
 
 - **xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
-- **Update a cluster**:
+- **xref:..//updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
```
Comment (jeana-redhat) [inline welcome/index.adoc:292]: This is working as-is, but idk if the portal is pickier so best to correct it :sweat_smile: 
```suggestion
- **xref:../updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
```

**Review by jeana-redhat — COMMENTED** [review]
Great efforts further refining and updating this page! Just found a few things to look at before merge.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: welcome/index.adoc (hunk 9 above)
Comment (dfitzmau) [reply welcome/index.adoc:87]: Nice catch. Updated.

File: welcome/index.adoc (hunk 10 above)
Comment (dfitzmau) [reply welcome/index.adoc:93]: Removed duplicates.

File: welcome/index.adoc (hunk 11 above)
Comment (dfitzmau) [reply welcome/index.adoc:97]: :heavy_check_mark:

File: welcome/index.adoc (hunk 12 above)
Comment (dfitzmau) [reply welcome/index.adoc:106]: "Allows" would be a form of anthropomorphism, so I'll stick with the original revision. Thanks.

File: welcome/index.adoc (hunk 13 above)
Comment (dfitzmau) [reply welcome/index.adoc]: :heavy_check_mark:

File: welcome/index.adoc (hunk 14 above)
Comment (dfitzmau) [reply welcome/index.adoc]: :heavy_check_mark:

File: welcome/index.adoc (hunk 15 above)
Comment (dfitzmau) [reply welcome/index.adoc:212]: https://www.redhat.com/en/about/press-releases/red-hat-introduces-red-hat-trusted-software-supply-chain

File: welcome/index.adoc (hunk 16 above)
Comment (dfitzmau) [reply welcome/index.adoc:288]: :heavy_check_mark:

File: welcome/index.adoc (hunk 15 above)
Comment (jeana-redhat) [reply welcome/index.adoc]: "Red Hat Trusted Software Supply Chain" then I guess? What a mouthful :dizzy_face:

Comment (dfitzmau) [reply welcome/index.adoc]: Sorry, I also pinged Gaurav Trivedi on Slack to confirm :pen:

File: welcome/index.adoc (hunk 12 above)
Comment (dfitzmau) [reply welcome/index.adoc]: NP. How about?

> This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.

Comment (jeana-redhat) [reply welcome/index.adoc]: Sounds good to me! Thanks :nerd_face:

File: welcome/index.adoc (hunk 15 above)
Comment (dfitzmau) [reply welcome/index.adoc]: PR https://github.com/openshift/openshift-docs/pull/70231 is planned for merging before the 4.15 GA. I'll create a subtask on my Welcome page Jira to comment in the link when the SSCS doc is published.

Comment (dfitzmau) [reply welcome/index.adoc]: For "details page", see [link](https://docs.openshift.com/pipelines/1.13/create/working-with-pipelines-web-console.html#op-creating-pipeline-templates-admin-console_working-with-pipelines-web-console).

**Comment by gtrivedi88** [comment]
@dfitzmau I've reviewed the software supply chain security content and it reads well. That you for adding and updating it.

File: welcome/index.adoc [hunk 17]
```diff
@@ -84,32 +84,33 @@ Explore the following {product-title} installation tasks:
 
 - **xref:../installing/installing_azure_stack_hub/preparing-to-install-on-azure-stack-hub.adoc#preparing-to-install-on-azure-stack-hub[Install a cluster on Azure Stack Hub]**: On Microsoft Azure Stack Hub, you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
 
-- **xref:../installing/installing_gcp/preparing-to-install-on-gcp.adoc#preparing-to-install-on-gcp[Install a cluster on GCP]**: On Google Cloud Platform (GCP) you can install {product-title} on installer-provisioned infrastructure or user-provisioned infrastructure.
+- **xref:../installing/installing_on_prem_assisted/installing-on-prem-assisted.html#using-the-assisted-installer_installing-on-prem-assisted[Installing {product-title} with the Assisted Installer]**: The Assisted Installer is an installation solution that is provided on the Red Hat {hybrid-console}. The Assisted Installer supports installing an {product-title} cluster on many platforms, but with a focus on bare metal, Nutanix, and {vmw-full} infrastructures.
```
Comment (dfitzmau) [inline welcome/index.adoc:87]: Storage PRs are open:

https://github.com/openshift/openshift-docs/pull/76847
https://github.com/openshift/openshift-docs/pull/75920
https://github.com/openshift/openshift-docs/pull/73710

**Pull request merged**